app/src/components/ui/organisms/Pomodoro/components.tsx

```diff
@@ -0,0 +1,35 @@
+import { useStore } from '@nanostores/react'
+
+import { handlePhase } from '@store/Pomodoro/actions'
+import { pomodoroStore } from '@store/store'
+import type { PhaseType } from '@store/types'
+
+import Button from '@atoms/Buttons/Button'
+
+import { BUTTON_TEXT } from './constants'
+import './styles.css'
+
+export const PhaseButtons = () => {
+  const { phase } = useStore(pomodoroStore.state)
+
+  const buttonPhases = ['pomodoro', 'break', 'longBreak'] as const
+
+  return (
+    <div className="pomodoro__buttons">
+      {buttonPhases.map((buttonPhase, index) => {
+        let styles = 'button '
+        styles = buttonPhase === phase ? 'button__chosenOption' : 'button__options'
```

Comment:
I think the names `pomodoro__button--selected` and `pomodoro__button` make more sense here for these 2 classes.

Although `pomodoro__button` would always be applied, and `pomodoro__button--selected` just for the selected button.